config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -0,0 +1,2678 @@
+# Autogenerated by genjobs.go please do not edit!
+# To modify these jobs first update their kubernetes/kubernetes sources
+# and/or genjobs.go, then run `hack/update-jobs.sh`
+presubmits:
+  kubernetes-security/kubernetes:
+  - agent: kubernetes
+    always_run: true
+    cluster: security
+    context: pull-security-kubernetes-bazel-build
+    labels:
+      preset-bazel-scratch-dir: "true"
+      preset-service-account: "true"
+    max_concurrency: 0
+    name: pull-security-kubernetes-bazel-build
+    rerun_command: /test pull-security-kubernetes-bazel-build
+    run_after_success:
+    - agent: kubernetes
+      always_run: false
+      cluster: security
+      context: pull-security-kubernetes-e2e-kubeadm-gce
+      labels:
+        preset-k8s-ssh: "true"
+        preset-service-account: "true"
+      max_concurrency: 8
+      name: pull-security-kubernetes-e2e-kubeadm-gce
+      rerun_command: /test pull-security-kubernetes-e2e-kubeadm-gce
+      run_if_changed: ^(cmd/kubeadm|build/debs).*$
+      skip_branches:
+      - release-1.10
+      - release-1.9
+      - release-1.8
+      skip_report: true
+      spec:
+        containers:
+        - args:
+          - --ssh=/etc/ssh-security/ssh-security
+          - --repo=github.com/kubernetes-security/$(REPO_NAME)=$(PULL_REFS)
+          - --upload=gs://kubernetes-security-prow/pr-logs
+          - --timeout=75
+          - --
+          - --gcs-shared=gs://kubernetes-security-prow/bazel
```

Comment:
we probably should also dump config.json flags here as well? (or when we migrate the actual presubmit jobs)

Reply:
can I do that in a follow up? this PR is already XXL and I wanted to make sure there was no real diff between the old generation method and the new, I had to significantly refactor how it works internally.

Reply:
https://www.diffchecker.com/yjeNhctE